src/extension/wps/src/main/java/au/org/emii/wps/NetcdfOutputProcess.java

```diff
@@ -111,7 +111,8 @@ public FileRawData execute(
 
             notifySuccess(callbackUrl, callbackParams);
 
-            return new FileRawData(outputFile, "application/zip", "zip");
+            FileInputStream fileInputStream = new FileInputStream(outputFile);
+            return new StreamRawData("application/zip", fileInputStream,  "zip");
```

Comment:
No point using a StreamRawData and writing to a temporary file as well.   Perhaps leave it as FileRawData for the moment.